src/applications/claims-status/actions/index.jsx

```diff
@@ -152,15 +152,17 @@ export function fetchClaimsSuccess(response) {
   };
 }
 
-export function pollRequest({
-  onError,
-  onSuccess,
-  pollingInterval,
-  request = apiRequest,
-  shouldFail,
-  shouldSucceed,
-  target,
-}) {
+export function pollRequest(options) {
```

Comment:
I stopped destructuring the options inline so that we can simply pass the options object to the delayed call on line 185. Manually recreating the options is prone to error. I added a `pollingExpiration` option and originally forget to pass it to the delayed call to this method.